en/04_Changelogs/5.0.0.md

```diff
@@ -136,6 +136,7 @@ This empowers advanced `Extension` functionality such as [Versioned::canPublish(
 ### Other new features {#other-features}
 
 - [DataObject::get_one()][api:SilverStripe\ORM\DataObject::get_one()] can now be called directly from subclasses of `DataObject` without passing in a class as the first argument (e.g. `SiteTree::get_one(filter: ['Title:startsWith' => 'About'])`).
+- The elemental [TopPage DataExtension](api:DNADesign\Elemental\TopPage\DataExtension) is applied to [BaseElement](api:DNADesign\Elemental\Models\BaseElement) and [ElementalArea](api:DNADesign\Elemental\Models\ElementalArea) by default, and the corresponding [SiteTreeExtension](api:DNADesign\Elemental\TopPage\SiteTreeExtension) is applied to [SiteTree](api:SilverStripe\CMS\Model\SiteTree) by default. This allows you to use `$TopPage` in your elemental block templates as a more performant alternative to `$Page` when referring to the page the element belongs to.
```

Comment:
This is incorrect for the way this has been implemented - those extensions are _not_ being applied, they've been completely replaced.

Reply:
We're now doing this via extensions again, so I'd say the docs here are correct